data_store/aws.py

```diff
@@ -86,32 +87,36 @@ def get_failed_file_key(failure_uuid: UUID) -> str:
     return file_key
 
 
-def get_file_metadata(
+def get_file_header(
     bucket_name: str,
     file_key: str,
-    return_full_head_data: bool = False,
+    formatted: bool = False,
 ) -> Dict[str, str]:
-    """Get metadata of a file stored in S3.
+    """Get header info of file stored in S3.
 
     :param bucket_name: string
     :param file_key: string
-    :param return_full_head_data: bool
-
-    :return: Metadata as a dictionary. Raises an exception if an error occurs.
+    :return: File header info as a dictionary. Raises an exception if an error occurs.
     """
+
     try:
         s3_response = _S3_CLIENT.head_object(Bucket=bucket_name, Key=file_key)
     except ClientError as error:
         if error.response["Error"]["Code"] == "404":
-            raise FileNotFoundError(
-                (f"Could not find file {file_key} in S3."),
-            ) from error
+            raise FileNotFoundError(f"Could not find file {file_key} in S3.") from error
+
         raise error
 
-    if return_full_head_data:
-        return s3_response
+    if formatted:
+        s3_response.update(
+            {
+                "ContentLength": get_human_readable_file_size(s3_response["ContentLength"]),
+                "ContentType": get_file_format_from_content_type(s3_response["ContentType"]),
+                "LastModified": s3_response["LastModified"].strftime("%d %B %Y"),
+            }
+        )
```

Comment:
I don't see anywhere that we use the unformatted response - do we just want to always format it and remove the `formatted` param from the signature?

Reply:
nit: I also find the `.update()` syntax a bit weird - curious why you prefer that over just doing assignments?

```python
s3_response['ContentLength'] = get_human_readable_file_size(s3_response["ContentLength"])
...
```

Reply:
I think I used `update` recently in another place, so now we are bffs. 
Happy to change it.

As for the `formatted` parameter. Happy to drop it too. My first instinct when I refactor is to try to maintain the former behavior.